translator/translate/otel/exporter/otel_aws_cloudwatch_logs/translator.go

```diff
@@ -56,8 +58,6 @@ func (t *translator) Translate(c *confmap.Conf) (component.Config, error) {
 	// Add more else if when otel supports log reading
 	if t.name == common.PipelineNameEmfLogs && t.isEmf(c) {
 		defaultConfig = defaultAwsCloudwatchLogsDefault
-	} else {
```

Comment:
Double checking that though we remove this, this currently gets used only by the emf pipeline and we are not adding new usages?